collectors/proc.plugin/proc_net_dev.c

```diff
@@ -741,11 +741,13 @@ int do_proc_net_dev(int update_every, usec_t dt) {
             char buffer[FILENAME_MAX + 1];
 
             snprintfz(buffer, FILENAME_MAX, path_to_sys_devices_virtual_net, d->name);
-            if(likely(access(buffer, R_OK) == 0)) {
+            if (likely(access(buffer, R_OK) == 0)) {
                 d->virtual = 1;
-            }
-            else
+                rrdlabels_add(d->chart_labels, "type", "virtual", RRDLABEL_SRC_AUTO);
```

Comment:
should we destroy labels on release?

Reply:
https://github.com/netdata/netdata/blob/b32ca44319e35eb38e5858730f2ea44ea2268926/collectors/proc.plugin/proc_net_dev.c#L315
https://github.com/netdata/netdata/blob/b32ca44319e35eb38e5858730f2ea44ea2268926/collectors/proc.plugin/proc_net_dev.c#L418